### PR TITLE
Buttons, outline buttons and icon buttons added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
also combined required colors in variables

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,176 @@
+.btn{
+    padding: 0.8rem;
+    margin: 1.5rem;
+    border-radius: 5px;
+    border: none;
+    letter-spacing: 0.1rem;
```
Comment (viru2001) [inline components/button/button.css]: for small measurements like this, you can use px instead of relative units

File: styles.css [hunk 2]
```diff
@@ -6,11 +6,24 @@ html, body{
...
+  --danger:#d13d30;
+  --danger-light:#c66c64db;
+  --alert:rgb(223, 123, 123);
+  --warning:rgb(255, 238, 0);
+  --warning-light:rgba(233, 233, 62, 0.823);
+  --info:#07d3fb;
+  --info-light:#8ed7e6e3;
+  --light:#dde5ec;
+  --dark:#1c1f23;
+  --light-green:rgb(36, 148, 128);
```
Comment (viru2001) [inline styles.css]: you can use rgb() or hex only to declare colors. don't use both types simultaneously.

**Review by munjalSaksham71 — COMMENTED** [review]
Looks good

File: styles.css (hunk 2 above)
Comment (vaibhav18matere) [reply styles.css]: got it, thanks!

**Review by vijaytembugade — APPROVED** [review]
Looks clean! I hope you can do changes asked by Viresh.

File: styles.css (hunk 2 above)
Comment (vaibhav18matere) [reply styles.css]: made required changes, thanks for valuable inputs !

**Comment by vaibhav18matere** [comment]
made required changes

**Pull request closed without merging**